### PR TITLE
feat: add setting for copy_commit_priority

### PR DESCRIPTION
**Pull request by olavloite**
Adds the setting 'spanner.copy_commit_priority' that can be used to set the Rpc priority for Commit requests for COPY operations. The default is MEDIUM to be consistent with Dataflow.

Fixes b/254330479

### CODE REVIEW

**Review by aseering — APPROVED** [review]
I'm once again a bit out of my depth in this codebase, but I like the objective and the UX of its implementation, and the code seems good insofar as I have the expertise to gauge such things.

File: src/main/java/com/google/cloud/spanner/pgadapter/session/CopySettings.java [hunk 1]
```diff
@@ -86,6 +88,19 @@ public int getCommitTimeoutSeconds() {
     return sessionState.getIntegerSetting("spanner", "copy_commit_timeout", 300);
   }
 
+  /** Returns the request priority for commits executed by COPY operations. */
+  public RpcPriority getCommitPriority() {
```
Comment (thiagotnunes) [inline src/main/java/com/google/cloud/spanner/pgadapter/session/CopySettings.java:92]: nit: `getCopyCommitPriority()`

**Review by thiagotnunes — COMMENTED** [review]
lgtm

File: src/main/java/com/google/cloud/spanner/pgadapter/session/CopySettings.java (hunk 1 above)
Comment (olavloite) [reply src/main/java/com/google/cloud/spanner/pgadapter/session/CopySettings.java:92]: I don't think I agree with that suggestion. The class is already called `CopySettings`, so the additional `Copy` in the method name feels redundant. It's also not in line with the other method names in the class.

Comment (thiagotnunes) [reply src/main/java/com/google/cloud/spanner/pgadapter/session/CopySettings.java:92]: I missed the class context, please ignore

**Review by thiagotnunes — APPROVED** [review]

**Pull request merged**